plugins/repo/subcommands/purge-cache

```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/env bash
+set -eo pipefail; [[ $DOKKU_TRACE ]] && set -x
+source "$PLUGIN_CORE_AVAILABLE_PATH/common/functions"
+
+repo_purge_cache() {
+  declare desc="deletes the contents of the build cache stored in the repository"
+  local cmd="repo:purge-cache"
+  [[ -z $2 ]] && dokku_log_fail "Please specify an app to run the command on"
+  local APP="$2"; local CACHE_DIR="$DOKKU_ROOT/$APP/cache"
+  verify_app_name "$APP"
+
+  find "$CACHE_DIR" -mindepth 1 -exec rm -rf {}
```

Comment:
Does this work? Check the apps `pre-delete` trigger. We mount a container to `rm` this in that case due to permissions.